screwdriver/api/api.go

```diff
@@ -223,6 +235,47 @@ func (c client) PostCommand(specPath string, commandSpec *util.CommandSpec) (*ut
 	return responseSpec, nil
 }
 
+func (c client) ValidateCommand(yamlString string) (*util.ValidateResponse, error) {
+	var (
+		body        *bytes.Buffer
+		contentType = "application/json"
+		err         error
+	)
+	body, err = writeValidateBody(yamlString)
+
+	uri, err := url.Parse(c.baseURL)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to parse URL on POST: %v", err)
+	}
+	uri.Path = path.Join(uri.Path, "validator/command")
+
+	responseBytes, statusCode, err := c.sendHTTPRequest("POST", uri.String(), contentType, body)
+	if err != nil {
+		return nil, fmt.Errorf("Post request failed: %v", err)
+	}
+
+	switch statusCode / 100 {
```

Comment:
I think it is difficult to use handleResponse because POS /validator/command does not return `command.Spec`. 
However I think it is better to collect this switch  in a single function, if it is not so hard.

Reply:
```go
responseFormat = new(util.ValidateResponse)
handleResponse(responseBytes, statusCode, responseFormat)
```
like this?

Reply:
I fixed handleResponse and use fixed handleResponse in ValidateCommand and httpRequest.